src/doc/style-guide/src/cargo.md

```diff
@@ -17,8 +17,7 @@ followed by the `description` at the end of that section.
 Don't use quotes around any standard key names; use bare keys. Only use quoted
 keys for non-standard keys whose names require them, and avoid introducing such
 key names when possible.  See the [TOML
-specification](https://github.com/toml-lang/toml/blob/master/versions/en/toml-v0.4.0.md#table)
-for details.
+specification](https://toml.io/en/v1.0.0#keys) for details.
```

Comment:
Why doesn't think link to https://toml.io/en/v1.0.0 ?

Reply:
Picked a link to this heading because the old link was to a heading as well, and this heading is the most direct translation. 
 The link is in the style guide section about keys and answers the question "what keys are non-standard and require quoting?".  Either link does the job.  I'm happy to change it if you think it's better to link to the document root rather than a section.

Reply:
Oh, in that case, sure.